### ISSUE TITLE
PSB (Large PSD) previews please - it's a must for designers

### ISSUE THREAD

**Issue by khrisf** [QL-Win/QuickLook#768]
Since I've used QuickLook since 2018 (moving from my mac), I was already wishing for PSB (large PSD) previews to get implemented~

It's 2020 and I'm a bit tired of waiting... so I'm here. LOL!

Anyhow, will this be added as a feature in the future?

Or is there a plugin that I could use to achieve this?

Thank you so much for an awesome product btw!

You really saved my relationship with Windows and got reunited coz of this app! ☺

**Comment by rabelux** [comment]
The dev doesn't have much time left for this project and so far noone has stepped in to contribute.
I guess waiting and hoping is all u can do

**Comment by mooflu** [comment]
ImageMagick supports psb, so I think it's just a matter of adding ".psb" extension to the ImageViewer plugin.

**Comment by rabelux** [comment]
@mooflu you're right, if the filetype is supported adding it to the list of extensions is enough.
Regarding your suggested and IMHO more elegant approach of dynamically matching supported filetypes:
Do you know that you can implement the feature in your fork, test it and start a pull-request? That way changes can be discussed, code can still be altered and @xupefei should have less workload implementing everything himself.
However I want to thank you for your suggestions, irrespective of how you decide! :slightly_smiling_face:

**Comment by bonjour-khris** [comment]
Looking forward to this feature! PSB previews pretty please.

**Comment by seung-pluto** [comment]
https://1drv.ms/u/s!AjeIEtkU1qTig-NnE5Y4NPhwCEk9rA?e=7c8BrD

Here, I rebuilt Quicklook.Plugin.ImageViewer simply with .psb added as a possible extension.  This was literally as simple as adding ".psb" in the right place.  I've tested it with multiple PSBs and it works (albeit slowly the huger the PSB is).  I will note, though, that PSB files are generally huge -- so you'll be left waiting for quite some time if you try to preview a multi-gigabyte file.
